config/manager/kustomization.yaml

```diff
@@ -1,3 +1,10 @@
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
```

Comment:
These comment lines make no sense here. You can remove them.
The last comment mentions that the namespace should have the same prefix. Have you checked if it is ok, that the namespace `kubevirt` does not have the `ssp-` prefix?